quasar.config.js

```diff
@@ -77,7 +77,8 @@ module.exports = configure(function (/* ctx */) {
       // analyze: true,
       env: {
         APP_NAME: 'Rectle',
-        GOOGLE_IDENTITY_CLIENT_ID: '680414393195-ikkj5vafetq5q67t499k60n7b1hpiuaf.apps.googleusercontent.com'
+        GOOGLE_IDENTITY_CLIENT_ID: '680414393195-ikkj5vafetq5q67t499k60n7b1hpiuaf.apps.googleusercontent.com',
+        BACKEND_URL: ctx.dev ? `http://localhost:8081/api/v1` : `https://rectle-service-dgcodfmmgq-lm.a.run.app/api/v1`
```

Comment:
Może wykorzystaj ten plugin: https://quasar.dev/quasar-cli-webpack/api-proxying

Reply:
A to będzie działać tylko na devie

Reply:
nie no chyba nie, wez przetestuj po prostu

Reply:
Ja bym kiedyś zrobił po prostu config/dev.env.ts i config/prod.env.ts i tam trzymał te API-URL.